Mathlib/Analysis/Normed/Group/Basic.lean

```diff
@@ -9,6 +9,7 @@ import Mathlib.Topology.Instances.Rat
 import Mathlib.Topology.MetricSpace.Algebra
 import Mathlib.Topology.MetricSpace.IsometricSMul
 import Mathlib.Topology.Sequences
+import Mathlib.Topology.Instances.NNReal
```

Comment:
This pulls in a *lot* of new dependencies, so we really shouldn't do this. `Analysis.Normed.Group.Basic` is too long already. Please start a new file `Analysis.Normed.Group.Lemmas` where you can put your new lemmas, and add a TODO to it saying we should split part of  `Analysis.Normed.Group.Basic` off into this file.